### PR TITLE
Add new quotes by Gaston Bachelard and Frank Smith.

### PR DESCRIPTION
**Pull request by saurav-jaat**

### CODE REVIEW

File: server/data/quotes.json [hunk 1]
```diff
@@ -220,7 +200,17 @@
...
+  },
+ {
+     "sentence": "Language is not a genetic gift, it is a social gift. Learning a new language is becoming a member of the club – the community of speakers of that language.",
+     "author": " Frank Smith"
+ },
+ {
+     "sentence": "A special kind of beauty exists which is born in language, of language, and for language.",
+     "author": "Gaston Bachelard"
+ }
+
```
Comment (jayehernandez) [inline server/data/quotes.json:214]: Please remove the new lines here so it’s cleaner!

Comment (saurav-jaat) [reply server/data/quotes.json:214]: Which new line?

Comment (jayehernandez) [reply server/data/quotes.json:214]: Lines 214 and 215

Comment (saurav-jaat) [reply server/data/quotes.json]: Done!!

**Comment by jayehernandez** [comment]
Thanks for contributing @saurav-jaat! Congrats on your first pull request.

@all-contributors please add @saurav-jaat for content

**Comment by saurav-jaat** [comment]
Thanks alot!! @jayehernandez

**Pull request merged**